GoogleSignIn/Sources/GIDAuthentication_Private.h

```diff
@@ -16,43 +16,16 @@
 
 #import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDAuthentication.h"
 
-#ifdef SWIFT_PACKAGE
-@import AppAuth;
-@import GTMAppAuth;
-#else
-#import <AppAuth/AppAuth.h>
-#import <GTMAppAuth/GTMAppAuth.h>
-#endif
-
 NS_ASSUME_NONNULL_BEGIN
 
 // Internal methods for the class that are not part of the public API.
-@interface GIDAuthentication () <GTMAppAuthFetcherAuthorizationTokenRefreshDelegate>
+@interface GIDAuthentication ()
 
 // A representation of the state of the OAuth session for this instance.
 @property(nonatomic, readonly) OIDAuthState *authState;
 
-#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
-// A string indicating support for Enterprise Mobility Management.
-@property(nonatomic, readonly) NSString *emmSupport;
-#endif // TARGET_OS_IOS && !TARGET_OS_MACCATALYST
-
 - (instancetype)initWithAuthState:(OIDAuthState *)authState;
 
-#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
-// Gets a new set of URL parameters that also contains EMM-related URL parameters if needed.
-+ (NSDictionary *)parametersWithParameters:(NSDictionary *)parameters
```

Comment:
These methods are used by another method of GIDAuthentication - `doWithFreshTokens:`. It will be moved into GIDGoogleUser in the next PR. Then we can delete GIDAuthentication class.